src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkWindows/Generated/Recipe.cs

```diff
@@ -370,30 +388,64 @@ private void ConfigureBeanstalkEnvironment(Configuration settings, string beanst
 
             if (settings.VPC.UseVPC)
             {
-                optionSettingProperties.Add(new CfnEnvironment.OptionSettingProperty
+                if (settings.VPC.CreateNew)
                 {
-                    Namespace = "aws:ec2:vpc",
-                    OptionName = "VPCId",
-                    Value = settings.VPC.VpcId
-                });
+                    if (AppVpc == null)
+                        throw new InvalidOperationException($"{nameof(AppVpc)} has not been set. The {nameof(ConfigureVpc)} method should be called before {nameof(ConfigureBeanstalkEnvironment)}");
 
-                if (settings.VPC.Subnets.Any())
-                {
+                    optionSettingProperties.Add(new CfnEnvironment.OptionSettingProperty
+                    {
+                        Namespace = "aws:ec2:vpc",
+                        OptionName = "VPCId",
+                        Value = AppVpc.VpcId
+                    });
+
+                    var vpcSubnets = new SortedSet<string>();
```

Comment:
Same comment as the Linux version about handling private and public subnets.

Reply:
Done